### DISCUSSION TITLE
Feature Request - TTY Viewer on Web Console

### DISCUSSION THREAD

**Discussion by ZanaGB** [aldostools/webMAN-MOD#1111] [Ideas]
Having a way to visualize the TTY without needing to install ProDG target tools would be interesting. Specially since there's no known ( to me, at least ) way to print the terminal output with telnet (sending specific commands to get it to print) or any simple tool that latches to port 1000. Considering there's views for many of the internal details, it could be useful to have a way to open an URL on your browser and just see the debug messages roll away.

**Comment by aldostools** [comment]
Hellow ZanaGB,
There is a compilation flag USE_DEBUG that sends raw text to port 38009 of a hardcoded IP address.
https://github.com/aldostools/webMAN-MOD/blob/master/main.c#L565-L573

That code added by DeanK for debugging the original webMAN. I don't use that method. I prefer to write the output to a file.
He used ssend if various places to display checkpoints and internal details like you said.

**Reply by ZanaGB** [reply]
Well, that could prove useful for what i'm looking for. I'll have to figure out at a later date if simply estabilishing a socket with something as crude as telnet will suffice to get a log.

Thanks for letting me know about the flag!
